packages/functional-tests/src/tests/font-test.ts

```diff
@@ -0,0 +1,221 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as MRE from '@microsoft/mixed-reality-extension-sdk';
+
+import { Test } from '../test';
+import { Vector3 } from '@microsoft/mixed-reality-extension-sdk';
+
+const options = {
+	font: Object.keys(MRE.TextFontFamily) as MRE.TextFontFamily[],
+};
+
+/** Defines an animation control field */
+interface ControlDefinition {
+	/** Decorative label for the control */
+	label: string;
+	/** Changes a property, and returns a result string */
+	action: (incr: number) => string;
+	/** Whether the control should be updated on a timer */
+	realtime?: boolean;
+	/** The actor who's text needs to be updated */
+	labelActor?: MRE.Actor;
+}
+
+// };
+/**
+ * Test the text api functionality
+ */
+export default class TextTest extends Test {
+	public expectedResultDescription = "Unicode Block and Font Test";
+	public interval: NodeJS.Timeout;
+	private assets: MRE.AssetContainer;
+
+	private textActors: MRE.Actor[] = [];
+
+	private textBlocks: any[] = [];
+	private currentList = 0;
+
+	private cycleOptions(): void {
+		for(const text of this.textActors) {
+			text.text.contents = this.textBlocks[this.currentList].displayString;
+		}
+	}
+	
+	public cleanup() {
+		clearInterval(this.interval);
+		this.assets.unload();
+	}
+
+	public createDisplayString(x: any) {
+		const unicodeChars: number[] = [];
+		for(let i = x.blockStart; i < x.blockEnd; ++i){
+			unicodeChars.push(i);
+
+			//Insert newline every 60 characters
+			if((i - x.blockStart) % 60 === 0) {
+				unicodeChars.push('\n'.charCodeAt(0));
+			}
+		}
+		x.displayString = String.fromCharCode(...unicodeChars);
+	}
+
+	public async run(root: MRE.Actor): Promise<boolean> {
+		this.assets = new MRE.AssetContainer(this.app.context);
+
+		this.textBlocks.push(
+			{blockStart: 0x20, blockEnd:0x07E, name:"Latin Basic"},
+			{blockStart: 0xA0, blockEnd: 0xFF, name:"Latin Suppliment"},
+			{blockStart: 0x100, blockEnd:0x017F, name:"Latin Extended - A"},
+			{blockStart: 0x3060, blockEnd:0x309F, name:"Japanese Hiragana/Katakana"},
+			{blockStart: 0x4E00, blockEnd:0x4E7F, name:"Chinese/Japanese/Korean Ideographs"});		
+
+		this.textBlocks.forEach((v,i,a) =>{ this.createDisplayString(v); });
+
+
+		const position = new Vector3(0,2.5,-.3);
```

Comment:
>Vector3 [](start = 23, length = 7)

Prefix with MRE and remove the extra include to keep it consistent with other tests

Reply:
Yeah I need to find the setting for these auto imports, every time I try the lazy way without  the MRE. it adds the import and I forget to double check.